### DISCUSSION TITLE
Rendering glitch

### DISCUSSION THREAD

**Discussion by apecollector** [google/model-viewer#3388] [Q&A]
I'm using model-viewer and for 99% of our models it works flawlessly. However there is a weird glitch where in some of our 3D models, the eyeballs show through the skin, especially when panning, yawing or zooming. This doesn't happen in https://threejs.org/editor/ or https://sandbox.babylonjs.com/ or blender / other 3D viewing software.

The model: https://cdn.themodz.dev/kids/models/3316.glb
The demo: https://themodz.dev/viewer/kids/3316

Example screenshot:

<img width="427" alt="Screen Shot 2022-04-25 at 5 44 42 PM" src="https://user-images.githubusercontent.com/100589991/165074277-bdf257e6-5d71-4419-90ea-d2bebffd00c3.png">

**Comment by elalish** [comment]
That looks like z-fighting, which is a precision issue and so it's affected by lots of minor camera parameter changes. Also, I'm seeing the same problem in the three.js editor: 
<img width="346" alt="image" src="https://user-images.githubusercontent.com/1649964/165143675-7d38b8c2-0a73-4a5e-b172-1107d5f11ac3.png">
How far behind the eyelids are the eye surfaces? Hopefully you can just push them back a little. If not this, may be a three.js transparency bug.

**Reply by apecollector** [reply]
Thanks, I didn't see it in three.js editor until I zoomed out, I'll ask our model artists about moving the eyes a little bit back and if that doesn't help I'll make an issue upstream with three.js.
